### DISCUSSION TITLE
How to upgrade to Nuxt 3 stable?

### DISCUSSION THREAD

**Discussion by DenFin** [nuxt/nuxt#16358] [Questions]
Really thrilled that Nuxt 3 is stable now!

How can we upgrade to the stable release from a RC? I have a project which uses RC 12. When I run `npx nuxi upgrade` it upgrades to RC 14.

**Comment by FSL89** [comment]
I ran **npm install -D nuxt** and it installed Nuxt 3.0.0 for me, I had RC 12 previously
